tests/TestEndpoints.js

```diff
@@ -16,12 +16,15 @@ export class TestEndpoints {
   constructor({implementation, tag}) {
     this.implementation = implementation;
     this.tag = tag;
-    this.verifier = implementation.verifiers.find(
+    try {this.verifier = implementation.verifiers.find(
```

Comment:
@PatStLouis what about doing something like `this.verifier = this.verifier?.find(...) || null;`?

Reply:
@BigBlueHat I'm assuming you meant something like:
```
this.verifier = implementation.verifiers?.find(
      verifier => verifier.tags.has(tag)) || null;
```

Just tried it and it worked, I prefer this as a prettier solution, thank you